server/xpub-model/entities/manuscript/index.test.js

```diff
@@ -140,6 +141,117 @@ describe('Manuscript', () => {
     })
   })
 
+  describe('get fileStatus()', () => {
+    describe('given there are no files', () => {
+      it('returns READY', () => {
```

Comment:
I've left the implementation like this when there's no files on the manuscript. 

It seems like there could be another state here like `NONE` but this should work as is. 

If people feel strongly about it, leave a 👍 and I'll raise a separate issue.

Reply:
I prefer it as written. 👍

Reply:
I prefer it as written. 👍

Reply:
I prefer it as written. 👍